tests/integration/tier1/changed-grub-file/test_valid_changed_grub.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function
+
+import fileinput
+
+from envparse import env
+
+
+targetline = "GRUB_CMDLINE_LINUX"
+
+
+def test_modify_grub_valid(convert2rhel):
+    for line in fileinput.FileInput("/etc/default/grub", inplace=True):
+        blockcmt = "\n# comment added by test\n"
+        inlinecmt_post = "# comment added by test"
+        whitespace = "     "
```

Comment:
I would move these variable definitions before the for loop, so they are not "re-defined" each loop.

Reply:
Sure, thanks for noting that!